extensions/wikia/UserLogin/templates/UserSignupSpecial_WikiaMobileSubmit.php

```diff
@@ -2,6 +2,3 @@
 <p class="wikia-terms signup-footer">
 	<?= wfMessage('prefs-help-terms')->parse() ?>
 </p>
-<p class="signup-footer login">
```

Comment:
Should the unused CSS for the ```login``` class be removed as well? It looks like this could be removed: https://github.com/Wikia/app/blob/dev/extensions/wikia/UserLogin/css/UserSignup.wikiamobile.scss#L116-L119